dist/truncate.js

```diff
@@ -412,7 +412,7 @@
 
       // Update HTML if provided, otherwise use the current html and restore
       // the truncated content to the original if it's currently present.
-      if (html) {
+      if (html !== undefined) {
```

Comment:
I usually prefer `typeof html !== 'undefined'` because in strict mode (`use strict`) accessing an undefined variable without using `typeof` will be an problem.